src/App.js

```diff
@@ -1,11 +1,13 @@
 import Footer from "./Components/footer/Footer";
+import Signin from "./Components/Signin/Signin";
 import "./app.css"
 function App() {
   return (
     <div className="page-container">
       <div className="content-wrap">
-    
+
       </div>
+      <Signin />
       <Footer />
     </div>
   );
```

Comment:
- [ ] Please pull the latest changes from the project, and the **Signin** component I think should move to the Homepage component which was created by your colleagues. Of course, this isn't your fault since the new changes came just minutes before you submitted them, but please update these requirements.